qutebrowser/mainwindow/mainwindow.py

```diff
@@ -145,6 +145,17 @@ class MainWindow(QWidget):
         _private: Whether the window is in private browsing mode.
     """
 
+    # Application wide stylesheets
+    STYLESHEET = """
```

Comment:
Why in the `MainWindow`? If they really should be application-wide, the correct place would probably be via `QApplication::setStyleSheet` - but why not just the next-closest object which is the `AbstractTab`?

Reply:
Application wide stylesheets break a test (we're expecting some warnings to be provided if we pass an invalid stylesheet, which won't work if we pass a stylesheet ourselves). If we add the stylesheet here, we only have one per window, as opposed to one per tab. I'm a little worried that if we have many tabs and the hint stylesheet changes, it might get a little slower. Also, there's no need to be doing this work per-tab (so there's no need to take any overhead on tab create for this).